src/design-guidelines/declutter/examples/tooltip-declutter.tsx

```diff
@@ -54,6 +54,10 @@ export class TooltipUsageExample {
                     </div>
                 </div>
             </div>,
+            <limel-example-do-do-not
+                doDescription="aa"
+                doNotDescription="aa"
```

Comment:
what is "aa" Kia? 😀

Reply:
Looks like this was a "work in progress" inside a commit, that a was later squashed and corrected.

Reply:
hm, I'm not sure if it's gone - looks like it what you got after squashing